docs/content/tutorial/step_09.ngdoc

```diff
@@ -1,144 +1,414 @@
 @ngdoc tutorial
-@name 9 - Filters
+@name 9 - Routing & Multiple Views
 @step 9
 @description
 
 <ul doc-tutorial-nav="9"></ul>
 
-In this step you will learn how to create your own custom display filter.
 
-* In the previous step, the details page displayed either "true" or "false" to indicate whether
-certain phone features were present or not. We have used a custom filter to convert those text
-strings into glyphs: ✓ for "true", and ✘ for "false". Let's see what the filter code looks like.
+In this step, you will learn how to create a layout template and how to build an application that
+has multiple views by adding routing, using an Angular module called {@link ngRoute ngRoute}.
```

Comment:
this form of the link will not format the ngRoute as code block. Is that what you want?